README.md

```diff
@@ -63,7 +63,15 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> https://ferizad.github.io/Portfolio/
```

Comment:
- [optional] Kindly remove this repeated app link to make your repo looks professional. ℹ️